plugin/mediawiki_editor.py

```diff
@@ -86,11 +86,12 @@ def site():
     else:
         httpauth = None
 
-    s = mwclient.Site((scheme, base_url()),
+    s = mwclient.Site(base_url(),
```

Comment:
If we update to this, it seems like people on versions prior to 0.10.0 will have their plugin break. We could consider logic that considers the version used:

```
version = mwclient.__dict__.get('__version__') or mwclient.__dict__.get('__ver__')
version_tuple = version.split('.')

if version_tuple >= [ '0', '10', '0' ]:
  # code for 0.10.0+
else:
  # code for <0.10.0

Reply:
Thanks for the conditionals. I've updated the commit accordingly.